README.md

```diff
@@ -96,7 +96,8 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 14 (2022)                         |  1532:028C  |
 | Razer DeathStalker V2 Pro (Wireless)          |  1532:0290  |
 | Razer DeathStalker V2 Pro (Wired)             |  1532:0292  |
-| Razer Ornata V3 X                             |  1532:0294  |
+| Razer Ornata V3 X (0294)                      |  1532:0294  |
+| Razer Ornata V3 X (02A2)                      |  1532:02A2  |
```

Comment:
Keep the device names here as they are on the pacakging, so I guess just "Razer Ornata V3 X". Also keep the list sorted

Reply:
Has been changed